### PR TITLE
Use applier cache

### PR DESCRIPTION
**Pull request by itdove**
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +88,80 @@ Check using the following command:
...
+```
+## Onboarding
+
+1. Create config secret on the external cluster to access the hub cluster:
+
+```bash
+echo '
+apiVersion: v1
+data:
+  kubeConfig: <base64 kubeconfig toward the hub>
```
Comment (chrisahl) [inline README.md:110]: We need to sync up with what `Local Development` section in README has which contradicts this
```oc apply -f hack/hubconfig.yaml
oc create secret generic mce-kubeconfig-secret --from-file kubeConfig # Expects a kubeconfig file named kubeConfig
```

File: README.md [hunk 2]
```diff
@@ -78,9 +88,80 @@ Check using the following command:
...
+kind: Secret
+metadata:
+  name: <secret_name>
+  namespace: <your_namespace>
+type: Opaque
```
Comment (chrisahl) [inline README.md:115]: Not sure where the secret data is being set?

File: README.md (hunk 1 above)
Comment (itdove) [reply README.md:110]: I guess we have to add the `-n`

**Review by robinbobbitt — APPROVED** [review]

**Pull request merged**